middleware/forwarded_test.go

```diff
@@ -0,0 +1,100 @@
+package middleware
+
+import (
+	"net/http"
+	"testing"
+)
+
+func TestGetSource(t *testing.T) {
+	type args struct {
```

Comment:
This struct appears to be adding bulk without adding any value

Reply:
Indeed it is removed.